### PR TITLE
1041 default for -dsannotation and -metatypteannotation is *

### PR DESCRIPTION
**Pull request by djencks**
Signed-off-by David Jencks <david_jencks@yahoo.com>

### CODE REVIEW

File: biz.aQute.bndlib/src/aQute/bnd/component/DSAnnotations.java [hunk 1]
```diff
@@ -35,10 +35,12 @@
 		inherit, felixExtensions, extender
 	};
 
+	private static final Parameters	DEFAULT_HEADER	= OSGiHeader.parseHeader("*");
+
 	public boolean analyzeJar(Analyzer analyzer) throws Exception {
 		Parameters header = OSGiHeader.parseHeader(analyzer.getProperty(Constants.DSANNOTATIONS));
```
Comment (bjhargrave) [inline biz.aQute.bndlib/src/aQute/bnd/component/DSAnnotations.java]: Instead of the DEFAULT_HEADER, do `analyzer.getProperty(Constants.DSANNOTATIONS, "*")`. 

header.size()==0 will match `-dsannotations: `.

File: biz.aQute.bndlib/src/aQute/bnd/metatype/MetatypeAnnotations.java [hunk 2]
```diff
@@ -28,11 +28,13 @@
 		nested
 	}
 
+	private static final Parameters	DEFAULT_HEADER	= OSGiHeader.parseHeader("*");
+
 	public boolean analyzeJar(Analyzer analyzer) throws Exception {
 		Map<TypeRef,OCDDef> classToOCDMap = new HashMap<TypeRef,OCDDef>();
 		Parameters header = OSGiHeader.parseHeader(analyzer.getProperty(Constants.METATYPE_ANNOTATIONS));
```
Comment (bjhargrave) [inline biz.aQute.bndlib/src/aQute/bnd/metatype/MetatypeAnnotations.java]: Instead of the DEFAULT_HEADER, do `analyzer.getProperty(Constants.METATYPE_ANNOTATIONS, "*")`. 

header.size()==0 will match `-metatypeannotations: `.

File: biz.aQute.bndlib.tests/src/test/BuilderTest.java [hunk 3]
```diff
@@ -357,6 +357,8 @@ public static void testClassQuery() throws Exception {
 			a.setExportPackage("test.component");
 			a.setProperty("testcases", "${sort;${classes;extending;junit.framework.TestCase;concrete}}");
 			a.setProperty("Test-Cases", "${testcases}");
+			a.setProperty("-dsannotations", "!*");
+			a.setProperty("-metatypeannotations", "!*");
```
Comment (bjhargrave) [inline biz.aQute.bndlib.tests/src/test/BuilderTest.java]: Also test for empty string.

**Comment by bjhargrave** [comment]
@djencks I reviewed this with Peter. Please updated for 3.0. Thanks.
